content/news/2023-09-18-gcb-2023/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Galaxy taking part @ German Conference on Bioinformatics (GCB) 2023
+date: '2023-09-18'
+tease: "From the 11th to the 14th of September, Engy Nasr and Paul Zierep from the Freiburg Galaxy team travelled to Hamburg to join and present their posters at the German Conference on Bioinformatics (GCB) 2023"
+location:
+  name: Hamburg, Germany
+tags: [conference, talk, bioinformatics, poster, pathogen detection]
+authors: engynasr
+supporters:
+- DECHEMA
+- ELIXIR
+components: true
+subsites: [all-eu]
+---
+
+From the 11th to the 14th of Semptember, Engy Nasr (I) and Paul Zierep from the Freiburg Galaxy team travelled to Hamburg to join and present our posters about the 'Accessible and scalable Galaxy pipelines for fast and easy (foodborne) Pathogen detection and tracking' and 'Microbiome data analysis using the Galaxy platform' at the [**German Conference on Bioinformatics (GCB) 2023**](https://gcb2023.de/).
+
+<twitter tweet="1702283085729996879"></twitter> 
+
+<p align="center">
+<img src="engy_and_paul.png" width="500" height="500">
+</p>
+
+The German Conference on Bioinformatics (GCB) is an annual, international conference devoted to all areas of bioinformatics and meant as a platform for the whole bioinformatics community. Recent meetings attracted a multinational audience of approx. 250 participants each year.
+
+This year’s talks focused on Multiomics analysis with Machine learning for the serve of the public health.
+
+On the first day; there were multiple worshops, 7 of them were in the morning and 5 of them were in the afternoon, participants had the chance to choose one of them to attend.
+
+On the second day; we had two keynotes and two talks sessions. At the end of the second talks session, we had the poster session, were 140 posters were presented. Engy's poster, the poster presented by me, was about my project **Foodborne Pathogen Dectection using direct Nanopore** and how is it important to implement it within Galaxy. I have presented my work and showed the Pathogen Detection workflow created in Galaxy along with its [**training material**](https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.html) and results.
+
+<p align="center">
+<img src="engy_poster.png" width="500" height="500">
+</p>
+
+Paul has also presented his poster about **Microbiome data analysis using the Galaxy platform** with an example of our current project pAllori, where we create Galaxy workflows to perform multi mircobiome data analysis applying machine learning algorthims to identify patients with Helicobacter pylori.
+
```

Comment:
During the poster sessions, we also had the chance to meet Anne Hartebrodt and learn about **Time shifting computation to reduce the carbon emissions of software**. This technique is particular interesting for the galaxy project, in order to improve our environmental footprint. Thanks to the contact we made during GCB, Anne will preset her work during the European galaxy days [**EGD**](https://galaxyproject.org/events/2023-10-egd/egd/).